get_involved.md

```diff
@@ -8,7 +8,7 @@ toc: false
 ## Meetings
 
 **Bug Triage** - every Tuesday and Friday 10:30 ET (either EST or EDT) in #pulp-dev on Freenode (IRC). Come and
-participate in real-time. See the [triage process](http://docs.pulpproject.org/dev-guide/contributing/bugs.html#triage-process) for
+participate in real-time. See the [triage process](http://docs.pulpproject.org/en/2.21/dev-guide/contributing/bugs.html#triage-process) for
```

Comment:
I wasn't sure if we want to link to Pulp 2 or Pulp 3, but if the latter, I can update this to:
https://docs.pulpproject.org/bugs-features.html#triage

Reply:
Yup, we want to update to Pulp 3.